stdlib/obj.ml

```diff
@@ -18,8 +18,9 @@ type t
 external repr : 'a -> t = "%identity"
 external obj : t -> 'a = "%identity"
 external magic : 'a -> 'b = "%identity"
-external is_block : t -> bool = "caml_obj_is_block"
 external is_int : t -> bool = "%obj_is_int"
+let is_block a = is_int a |> not
+  [@@ocaml.inline]
```

Comment:
We should probably mark `double_field` and `set_double_field` as well for consistency. Given that they follow exactly the same pattern as `is_block` (small wrapper around a %-primitive) it might not be clear to readers why one is marked as `[@@inline]` but not the others